src/main/java/objects/inventory/Item.java

```diff
@@ -0,0 +1,142 @@
+package objects.inventory;
+
+public class Item {
+    /*
+    A class for Item. Includes getter and setter for all attributes.
+    Attributes:
+    level: The level of player.
+    type: The type of item (Armor, Weapon, Potion).
+    name: The name of item.
+    ability: The ability of item.
+    price: The price of item.
+
+     */
+    public enum Type{
+        Armor,
+        Weapon,
+        Potion
+    }
+    private int level;
+    private Type type;
+    private String name;
+    private String ability;
+    private int price;
+
+    public Item(int level, Type type) {
+        /*
+        Initializes a new Item with level, type, an empty name, an ability, and 0 price.
+         */
+
+        this.level = level;
+        this.type = type;
+        setAttributes();
+    }
+
+    public void setAttributes(){
+        /*
+        Set attributes name, ability, and price based on the level of player and the type of item.
+        THIS will be modified after discussing with battle system.
+         */
+        switch (type){
```

Comment:
Code smell - switch statement. This links to the big problem about item class which is that its not extendable.